privval/sc_file.go

```diff
@@ -111,6 +111,7 @@ func (pv *SCFilePV) run() {
 
 			var err error
 			if pv.SecretConn, err = connection.RetryDial(
+				config.Dir(),
```

Comment:
Is there a reason for this line?  the returning string is not used

Reply:
Not sure why you think it's not used. The config directory passed into RetryDial is used to locate the conn.key file when using a TCP connection between SignCTRL and the validator.

Reply:
Ah, I didn't see that it's an argument. Thought we were in the if block, sorry.